.circleci/config.yml

```
@@ -0,0 +1,20 @@
+version: 2.1
+workflows:
+  init_and_validate:
+    jobs:
+      - install_terraform_and_run_test
+jobs:
+  install_terraform_and_run_test:
+    docker:
+      - image: cimg/go:1.16
+    steps:
+      - checkout
+      - run: 
+          name: Install Terraform
+          command: curl -LO https://github.com/hashicorp/terraform/archive/refs/tags/v0.12.31.tar.gz && tar xf v0.12.31.tar.gz && cd terraform-0.12.31 && go install
+      - run:
+          name: Print Terraform version
+          command: terraform -version
```

Can we try this with our terraform CI image?
```suggestion
      - image: quay.io/astronomer/ci-terraform:2021-08
    steps:
      - checkout
      - run:
          name: Print Terraform version
          command: terraform-0.12 -version
```

---

I can check if CircleCI allows us to combine images, otherwise we'll need a step to install Go.

---

Or include Go in our Terraform image.

---

We can add golang to our terraform ci image if it is needed. The goals of using the internal ci-image are to make a consistent ci environment that we can use in all workflows without having to copy/paste setup steps, for CI to focus on testing the code and not break when installation of the testing setup fails, and for us to be able to easily upgrade all test environments by simply upgrading the used ci-image.